specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2018-09-01/BackupLocations.json

```diff
@@ -288,6 +334,67 @@
           "type": "string"
         }
       }
+    },
+    "PruneOption": {
+      "description": "Prune backup store operation option.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Operation option.",
+          "$ref": "#/definitions/PruneBackupStoreOperationOptionModel",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "PruneBackupStoreOperationType": {
+      "description": "Prune backup store operation type.",
+      "type": "string",
+      "enum": [
+        "Default",
+        "DryRun"
+      ],
+      "x-ms-enum": {
+        "name": "PruneBackupStoreOperationType",
+        "modelAsString": false
+      }
+    },
+    "PruneBackupStoreOperationOptionModel": {
+      "type": "object",
+      "description": "Prune backup store operation model.",
+      "properties": {
+        "OperationType": {
+          "description": "Operation type.",
+          "$ref": "#/definitions/PruneBackupStoreOperationType"
+        }
+      }
+    },
+    "PruneModel": {
+      "description": "Properties for a prune list.",
+      "properties": {
+        "pathsToDelete": {
+          "description": "Garbage file list.",
+          "type": "array",
+          "items": {
+              "type": "string"
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "PruneList": {
+      "type": "object",
+      "description": "List of files to delete.",
+      "properties": {
+        "properties": {
+          "description": "Holds information for a backup.",
+          "$ref": "#/definitions/PruneModel"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "Backup.json#/definitions/Resource"
```

Comment:
the response of a post action should contain information about the result of the action, it should not return properties of the resource it is acting upon - that can be retrieved via a GET on the resource

Reply:
Sorry I made a mistake in branch management. This comment is resolved.